content/docs/concepts/statistics/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Statistics Service
+description: Provide usage statistics about a Keptn installation and optionally display them on a Swagger dashboard.
+
+weight: 10
+icon: setup
+---
+
+
+The Statistics Service provides usage statistics about a Keptn installation.
+You can access the service using the Keptn API
+and view the results using the Swagger UI.
+
+## Generate Swagger doc from source
+
+First, install the `go` modules with the following commands:
+
+```
+go get -u github.com/swaggo/swag/cmd/swag
+go get -u github.com/swaggo/gin-swagger
+go get -u github.com/swaggo/files
+```
+
+To generate new source that updates the `swagger.yaml` file with new endpoints or models, execute the following:
+
+```
+swag init
+```
+
+## How to use the service
+
+Access the Statistics Service through the Keptn API under the `statistics` path.
+For example:
+
+```
+http://keptn-api-url.com/api/statistics
+```
+
+Use the following URL to view the swagger UI for the service:
+
+```
+http://keptn-api-url.com/api/swagger-ui/?urls.primaryName=statistics
+```
+
+To browse the API docs, open the Swagger docs in your [browser](http://localhost:8080/swagger-ui/index.html).
```

Comment:
Apparently this is a link to the local docs but it is a broken link if one is reading this on a system that is not running Keptn with the Statistics Service configured.  How can it be reworded?

Reply:
```suggestion
To browse the API docs, open the Swagger docs in your browser, e.g. `${KEPTN_URL}/swagger-ui/index.html`.
```

Reply:
In my environment - the URL is `/api/swagger-ui/`  -- it has the API in the path and index.html is not required to list.

Reply:
When we figure out how to resolve this issue here, I will generate a PR for https://github.com/keptn/keptn/blob/master/statistics-service/README.md  to fix the problem there until we de-duplicate the README text.